src/Alliteration.php

```diff
@@ -0,0 +1,36 @@
+<?php
+namespace Nubs\RandomNameGenerator;
+
+/**
+ * Defines the standard interface for all the random name generators.
```

Comment:
I'll want to get the phpdocs updated in here to mirror those of Vgng's.  Feel free to take a shot at it or I can do it.  I'll probably also update this to use the same randomizer so that the tests can be exact with predefined random results.